### PR TITLE
 join nodes concurrently to a cluster

### PR DESCRIPTION
**Pull request by AlonMagrafta**
https://redislabs.atlassian.net/browse/RED-46876
This can get to doc right away.

### CODE REVIEW

File: content/rs/administering/adding-node.md [hunk 1]
```diff
@@ -77,6 +77,12 @@ To add a node in RS:
 The node is added to the cluster. You can see it in the list of nodes in the
 cluster.
 
+{{< note >}}In case of adding multiple nodes to the clutter,we recommend
```
Comment (guyco-redis) [inline content/rs/administering/adding-node.md:80]:
```suggestion
{{< note >}}In case of adding multiple nodes to the cluster, we recommend
```

**Review by guyco-redis — APPROVED** [review]

**Comment by bmansheim** [comment]
@AlonMagrafta Seems to me that this note should be at the beginning of the process rather than the end. Do my changes look OK to you?

**Comment by AlonMagrafta** [comment]
@bmansheim let's hold it (there are some development and tests around it to understand some things there on my end)

**Comment by AlonMagrafta** [comment]
@bmansheim np with your rearrangement. Let's merge it

**Pull request merged**